src/pages/Shop/SideMenuList/SideMenu/SideMenu.scss

```
@@ -0,0 +1,19 @@
+.sideMenuGroup {
+  .sideMenuTitle {
+    padding: 10px 0;
+    a {
+      text-decoration: none;
+      font-weight: 500;
+      color: black;
+      .sideMenuCaret {
+        padding-left: 10px;
+      }
+    }
+  }
+  ul {
+    li {
+      padding: 10px 20px;
+      color: rgb(113, 113, 113);
+    }
+  }
+}
```

- 가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.
- nesting은 필요한 만큼만 사용해주세요. 스타일이 없는데 불필요하게 셀렉터 추가할 필요 없습니다!
```suggestion
.sideMenuGroup {
  .sideMenuTitle {
    padding: 10px 0;
    
    a {
      text-decoration: none;
      font-weight: 500;
      color: black;
      .sideMenuCaret {
        padding-left: 10px;
      }
    }
  }
  
  li {
    padding: 10px 20px;
    color: rgb(113, 113, 113);
  }
}
```

---

넵 반영하겠습니다!